src/components/Footer.tsx

```diff
@@ -213,20 +201,78 @@ const Logo = styled.div`
 const FooterLinkGroups = styled.div`
 	display: flex;
 	flex: 1;
+	width: 100%;
+
+	@media(max-width:768px){
+		flex-wrap: wrap;
+	}
 `;
 
 const FooterLinkGroup = styled.div`
-	flex: 1;
+	// flex: 1;
 	display: flex;
 	flex-direction: column;
-	gap: 2rem;
+	gap: 3rem;
+	width: 50%;
+
 	> h5 {
 		margin: 0;
 		font-size: 1.2rem;
 	}
+	@media(max-width:768px){
+		width: 50%;
+		margin-bottom: 4rem;
+	}
+
 `;
 
 const SocialsContainer = styled.div`
 	display: flex;
 	gap: 0.5rem;
 `;
+
+const LogoInfoWrapper = styled.div`
+	display: flex;
+	flex-rap: wrap;
+	gap: 5rem;
+	flex: 1;
+	margin-top: 4rem;
+
+
+	@media(max-width:768px){
```

Comment:
We need to define the breakpoints as variables: say mobile-, tablet- and desktopbreakpoint.

Reply:
Also use `max-width` as less as possible. To achieve a mobile first appproach. A solution is to put the content of the this query as the default, which would probably make it one col already. Then add the `display:flex` when the screen width get lager than your beakpoint.

Reply:
All Max-width have been corrected.